bandit/src/query-lambda/dynamo.ts

```diff
@@ -35,9 +35,9 @@ function buildDynamoRecord(
 	startTimestamp: string
 ): TestSample {
 	const variants = rows.map((row) => ({
-		variantName: row.variantName,
-		annualisedValueInGBP: row.avGbp,
-		annualisedValueInGBPPerView: row.avGbpPerView,
+		variantName: row.variant_name,
+		annualisedValueInGBP: row.sum_av_eur,
+		annualisedValueInGBPPerView: row.sum_av_eur_per_view,
```

Comment:
Currently we're using the EUR values in fields with GBP in the name.
How about adjusting the query to do the EUR -> GBP conversion?

This example might be useful: https://github.com/guardian/data-platform-models/blob/25c4bfea2028418dc3800281b7232a2d2d085688/dbt/models/datalake/contributions/contributions.sql#L35
It joins to both the EUR rate and the EUR -> GBP rate

Reply:
yes, I have updated the query and related files to do the EUR->GBP Conversion